server/config/index.js

```diff
@@ -1,5 +1,13 @@
 require('dotenv').config()
 
+const fs = require('fs')
+
+let graphiqlQueryFileContent = ''
+
+if (process.env.GRAPHIQL_QUERY_FILE) {
+  graphiqlQueryFileContent = fs.readFileSync(process.env.GRAPHIQL_QUERY_FILE).toString()
```

Comment:
two minor nits:

* you can pass `'utf8'` as a second argument to `fs.readFileSync` and a string will be returned (as opposed to a buffer)
* perhaps we should surround this in try/catch to avoid crashing when a file doesn't exist